spec/system/admin/reports/payments_report_spec.rb

```diff
@@ -97,4 +97,7 @@
       ].join(" "))
     end
   end
+  def i18n_translate(translation_key, options = {})
```

Comment:
I'm wondering where this method is used actually?

Reply:
Looks like we forgot to remove that function from a previous version. 

We created those 3 specs under spec/lib/reports/payments/* as simple unit tests to confirm that the translation worked. 

We are also working to pass the spec that we are failing.  The output for the test case we are failing is a little confusing. 
On line 38 we are failing because the actual output is printing some sort of list of translations instead of one translated payment state. 
The bug on line 73 is easy to fix since our expected payment state is incorrect. However, we do not understand why it says the output is "credit owed" when the orders are all declared as complete. Is it possible to complete an order that has not been paid yet? 
I wonder if instead of expecting a string like "completed" or "credit owed" we should be using something like order.payment_state under our expect clause?

I am not sure why running locally gives us errors. We are using docker and running the test cases takes a very long time and a high percentage fail even though our pull request says we only fail in two places. My guess is that some packages are not installed or installed improperly. It is very hard or us to debug why our test cases are not passing locally so we are trying to use pull request feedback.